stream.go

```diff
@@ -59,43 +59,11 @@ type StreamDesc struct {
 
 // Stream defines the common interface a client or server stream has to satisfy.
 //
-// All errors returned from Stream are compatible with the status package.
-type Stream interface {
-	// Context returns the context for this stream.  If called from the client,
-	// Should be done after Header or RecvMsg.  Otherwise, retries may not be
-	// possible to perform.
-	Context() context.Context
-	// SendMsg is generally called by generated code. On error, SendMsg aborts
-	// the stream and returns an RPC status on the client side. On the server
-	// side, it simply returns the error to the caller.
-	//
-	// SendMsg blocks until:
-	//   - It schedules m with the transport, or
-	//   - The stream is done, or
-	//   - The stream breaks.
-	//
-	// SendMsg does not wait for an ack. An untimely stream closure
-	// can result in any buffered messages along the way (including
-	// those in the client-side buffer that comes with gRPC by default)
-	// being lost. To ensure delivery, users must call RecvMsg until
-	// receiving an EOF before closing the stream.
-	//
-	// It's safe to have a goroutine calling SendMsg and another
-	// goroutine calling RecvMsg on the same stream at the same
-	// time. It is not safe to call SendMsg on the same stream
-	// in different goroutines.
-	SendMsg(m interface{}) error
-	// RecvMsg blocks until it receives a message or the stream is
-	// done. On client side, it returns io.EOF when the stream is done. On
-	// any other error, it aborts the stream and returns an RPC status. On
-	// server side, it simply returns the error to the caller.
-	// It's safe to have a goroutine calling SendMsg and another goroutine calling
-	// recvMsg on the same stream at the same time.
-	// But it is not safe to call RecvMsg on the same stream in different goroutines.
-	RecvMsg(m interface{}) error
-}
+// DEPRECATED: See ClientStream and ServerStream documentation instead.
```

Comment:
Nit: "Deprecated" (case) - it actually matters to tools like staticcheck.

Reply:
Done.